src/components/navigation/Navbar.tsx

```diff
@@ -1,12 +1,33 @@
 import React from "react";
 import { Link } from "react-router-dom";
+import "./Navbar.scss";
+import { useState } from 'react'
 
 export const Navbar: React.FunctionComponent = () => {
+
+  const [isExpanded, setIsExpanded] = useState(false)
+
+  function toggleMenu(){
+    setIsExpanded(current => !current)
+  }
+
   return <nav>
-    <ul>
-      <li><Link to="/">Home</Link></li>
-      <li><Link to="/rovers">Rovers</Link></li>
-      <li><Link to="/quiz">Quiz</Link></li>
-    </ul>
+
+    <div className="navbar">
+      <button onClick = {() => toggleMenu()}>
+      
+        <span className = {`navbar__spanone ${isExpanded ? "burgerexpanded" : "burgercollapsed"}`}></span>
+        <span className = {`navbar__spantwo ${isExpanded ? "burgerexpanded" : "burgercollapsed"}`}></span>
+        <span className = {`navbar__spanthree ${isExpanded ? "burgerexpanded" : "burgercollapsed"}`}></span>
```

Comment:
Prefer `kebab-case`, so `burger-expanded` and `burger-collapsed` (but you may be able to make it cleaner by either just using `expanded` and `collapsed`, or full BEM with `navbar__hamburger-line--expanded` etc)